File: VotingModelLibrary/Models/PollingStation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace VotingModelLibrary.Models
{
    public class PollingStation
    {
        [Key]
        public int PollingStationId { get; set; }
        public int ElectionId { get; set; }
        public Election Election { get; set; }

        public string Name { get; set; }

        [Display(Name = "Additional Info")]
        public string AdditionalInfo { get; set; }
        public string Address { get; set; }

        [Display(Name = "Wheelchair Info")]
        public string WheelchairInfo { get; set; }

        [Display(Name = "Parking Info")]
        public string ParkingInfo { get; set; }

        [Display(Name = "Parking Info")]
        public string WashroomInfo { get; set; }

        [Display(Name = "General Access Info")]
        public string GeneralAccessInfo { get; set; }
        public double Latitude { get; set; }
        public double Longitute { get; set; }
    }
}

File: Spa/ClientApp/src/app/models/pollingstation.ts
export class PollingStation {
  pollingStationId  : number;
  name              : string;
  additionalInfo    : string;
  latitude          : number;
  longitute         : number; //TEMPORARILY MISSPELLED TO MATCH API, FIX WHEN API IS FIXED
  electionId        : number;
  address           : string;
  wheelchairInfo    : string;
  parkingInfo       : string;
  washroomInfo      : string;
  generalAccessInfo : string;


  constructor(
    pollingStationId: number,
    name: string,
    additionalInfo: string,
    latitude: number,
    longitute: number, //TEMPORARILY MISSPELLED TO MATCH API, FIX WHEN API IS FIXED
    electionId: number,
    address: string,
    wheelchairInfo: string,
    parkingInfo: string,
    washroomInfo: string,
    generalAccessInfo: string,
  ) {
    this.pollingStationId   = pollingStationId;
    this.name               = name;
    this.additionalInfo     = additionalInfo;
    this.latitude           = latitude;
    this.longitute          = longitute; //TEMPORARILY MISSPELLED TO MATCH API, FIX WHEN API IS FIXED
    this.electionId         = electionId;
    this.address            = address;
    this.wheelchairInfo     = wheelchairInfo;
    this.parkingInfo        = parkingInfo;
    this.washroomInfo       = washroomInfo;
    this.generalAccessInfo = generalAccessInfo;
  }
}
